src/app/CASESessionManager.cpp

```diff
@@ -28,9 +28,15 @@ CHIP_ERROR CASESessionManager::FindOrEstablishSession(PeerId peerId, Callback::C
 
     bool nodeIDWasResolved = (mConfig.dnsCache != nullptr && mConfig.dnsCache->Lookup(peerId, resolutionData) == CHIP_NO_ERROR);
 
+    ChipLogDetail(CASESessionManager,
+                  "FindOrEstablishSession: PeerId = CF" ChipLogFormatX64 ":N" ChipLogFormatX64 ", NodeIdWasResolved = %d",
```

Comment:
I think doing the "NNNN-MMMM" thing that people expect for compressed fabric and node id would make the most sense here...

Reply:
Yes, agree. `CF` is also confusing since it's hex!